src/templates/expressions/fields.ts

```diff
@@ -23,124 +24,18 @@ ${toTsExpressions(this.fileFields)}
     }
 }
 
-export class StringField implements TsExpression {
+export class TSDefinedField implements TsExpression {
```

Comment:
`TS` might be `Ts` (`s` is lowercase)? If you adjust the class name to other TypeScript-related variables like `TsExpression`.